libs/platform/src/lib/components/menu-button/menu-button.component.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { MenuButtonComponent } from './menu-button.component';
+import { ButtonModule } from '@fundamental-ngx/core';
+
```

Comment:
I dont see any tests.

Reply:
will be writing test cases.

Reply:
included test cases for type and disabled menu buttons.